setup.py

```diff
@@ -61,7 +61,7 @@
     "invenio-indexer>=1.1.1",
     "invenio-records>=1.3.2",
     "invenio-rest>=1.2.1",
-    # "flask-resources", # FIXME: Currently in dev
+    "flask-resources~=0.2.0",
```

Comment:
Moderate: Since we are under heavy development. Shall we just lower pin until first? We ahve been doing that in invenio-rdm-records, invenio-app-rdm, etc. (https://github.com/inveniosoftware/invenio-rdm-records/blob/master/setup.py#L78)

```suggestion
    "flask-resources>=0.2.0,<1.0.0",
```

Reply:
@ppanero We do test `lower_requirements` so a bump most of the time will do be needed. I am ok with both options :)